scipy/special/_c99compat.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Portable versions of some C99 functions. In cmath they are only
+ * available in C++11 and the npy_math versions don't work well (they
+ * get undef'd by cmath; see gh-5689). Implementations based on
+ * npy_math.h and friends.
+ */
+#ifndef C99COMPAT_H
+#define C99COMPAT_H
+
+#include <Python.h>
+#include <numpy/numpyconfig.h>
+
+
+int
+sc_signbit(double x)
```

Comment:
I didn't look into it too carefully, but I assumed it was this issue: https://github.com/scipy/scipy/issues/5689#issuecomment-170259179. (The CI build complained about not being able to find `signbit`.)